launcher/Application.cpp

```diff
@@ -687,8 +687,16 @@ Application::Application(int &argc, char **argv) : QApplication(argc, argv)
                 m_settings->reset("PastebinCustomAPIBase");
             }
         }
-        // meta URL
-        m_settings->registerSetting("MetaURLOverride", "");
+        {
+            // Meta URL
+            m_settings->registerSetting("MetaURLOverride", "");
+
+            QUrl metaUrl = m_settings->get("MetaURLOverride").toString();
```

Comment:
```suggestion
            QUrl metaUrl(m_settings->get("MetaURLOverride").toString());
```

this is both more explicit and makes more sense